queries.py

```diff
@@ -0,0 +1,127 @@
+MAPPING = """
+PREFIX apf: <http://jena.hpl.hp.com/ARQ/property#>
+PREFIX fx: <http://sparql.xyz/facade-x/ns/>
+PREFIX premis: <http://www.loc.gov/premis/rdf/v3/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xyz: <http://sparql.xyz/facade-x/data/>
+
+CONSTRUCT {
+    ?ont_iri a skos:ConceptScheme ;
+                  skos:prefLabel "Meemoo event types thesaurus" ;
```

Comment:
Can we prevent hardcoding this ? perhaps by using the tabname? Or a CLI parameter?

Reply:
I'll add it in as a CLI parameter!